app/linux_notification_executor.rb

```diff
@@ -19,9 +18,15 @@ def execute(n)
           icon = 'dialog-error'
       end
 
-      hello = Notify::Notification.new(n.header, @simple ? n.body : n.generate_html_body, icon)
+      hello = Notify::Notification.new(n.header, @simple ? strip_complex_tags(n.body) : n.generate_html_body, icon)
+      puts "Stripped: #{strip_complex_tags(n.body)}"
```

Comment:
Tak na prawdę to zapomniałem usunąć flagę 'a' z git commit... ale tak, to było do odpluskwiania